specification/resourceconnector/resource-manager/Microsoft.ResourceConnector/stable/2022-10-27/appliances.json

```diff
@@ -311,40 +346,52 @@
             "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
           },
           {
-            "$ref": "#/parameters/ResourceNameParameter"
+            "name": "resourceName",
+            "in": "path",
+            "description": "Appliances name.",
+            "required": true,
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 63,
+            "pattern": "^[a-zA-Z0-9]$|^[a-zA-Z0-9][-_a-zA-Z0-9]{0,61}[a-zA-Z0-9]$"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The updatable fields of an existing Appliance.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/patchableAppliance"
+            }
```

Comment:
the original swagger shows as below, where the x-ms-client-flatten information is missing in new schema:
```
{
            "name": "parameters",
            "in": "body",
            "required": true,
            "x-ms-client-flatten": true,
            "schema": {
              "$ref": "#/definitions/patchableAppliance"
            },
            "description": "The updatable fields of an existing Appliance."
}
```
this change caused sdk breaking change:
<img width="729" height="127" alt="image" src="https://github.com/user-attachments/assets/83035900-6df5-47cc-a7cc-8e78e34aef15" />

Reply:
<img width="933" height="350" alt="image" src="https://github.com/user-attachments/assets/eaed8f67-632c-4eb9-966b-ef2962982f78" />

Reply:
https://github.com/Azure/typespec-azure/issues/3416